src/main/java/wooteco/subway/service/LineService.java

```diff
@@ -0,0 +1,43 @@
+package wooteco.subway.service;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import wooteco.subway.dao.LineDaoImpl;
+import wooteco.subway.domain.Line;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.exception.LineNotFoundException;
+
+@Service
+public class LineService {
+
+    private final LineDaoImpl lineDaoImpl;
+
+    public LineService(final LineDaoImpl lineDaoImpl) {
+        this.lineDaoImpl = lineDaoImpl;
+    }
```

Comment:
```suggestion
    private final LineDao lineDao;

    public LineService(final LineDao lineDao) {
        this.lineDao = lineDao;
    }
```
여길 보고 느낀건데 어차피 구체클래스를 주입받을 거라면 인터페이스가 필요할 거 같진 않네요.

Reply:
해당 부분 수정하였습니다!.  감사합니다 범블비!